tensorly/backend/mxnet_backend.py

```diff
@@ -57,12 +57,20 @@ def reshape(tensor, shape):
         return nd.reshape(tensor, shape)
 
     def solve(self, matrix1, matrix2):
+        # TODO: Replace this as soon as mxnet.np is stable.
```

Comment:
I don't use MXNet so I'm less familiar with its state, but `MXNet.np` looks promising. If it's feasible to switch over from the `ndarray` interface it would also remove most of the backend-specific code.